dnscontrol/creds.json

```diff
@@ -0,0 +1,7 @@
+{
+  "cloudflare": {
+    "TYPE": "CLOUDFLAREAPI",
+    "accountid": "e4a2db52c495db230973c839a0699ae1",
+    "apitoken": "$CLOUDFLARE_API_TOKEN"
```

Comment:
I thought we could use our own tokens instead of keeping the super admin account token in sops but don't feel strongly about it.